tests/src/test/java/alluxio/server/ft/journal/TriggeredCheckpointTest.java

```diff
@@ -67,6 +67,42 @@ public void ufsJournal() throws Exception {
     }
   }
 
+  @Test
+  public void ufsJournalWithParallelBackupRocksDb() throws Exception {
+    int numFiles = 100;
+    MultiProcessCluster cluster = MultiProcessCluster
+        .newBuilder(PortCoordination.TRIGGERED_UFS_CHECKPOINT)
+        .setClusterName("TriggeredUfsCheckpointTest")
+        .addProperty(PropertyKey.MASTER_JOURNAL_TYPE, JournalType.UFS)
+        .addProperty(PropertyKey.MASTER_JOURNAL_CHECKPOINT_PERIOD_ENTRIES, numFiles)
+        .addProperty(PropertyKey.MASTER_METASTORE, "ROCKS")
+        .addProperty(PropertyKey.MASTER_PARALLEL_BACKUP_ROCKSDB, true)
+        .addProperty(PropertyKey.MASTER_PARALLEL_BACKUP_ROCKSDB_THREAD_POOL_SIZE, 5)
+        .setNumMasters(1)
+        .setNumWorkers(1)
+        .build();
+    try {
+      cluster.start();
+      cluster.waitForAllNodesRegistered(20 * Constants.SECOND_MS);
```

Comment:
let's wait for 1s or more so this test doesn't become flaky

Reply:
The setting of 20s is inherited from other tests in TriggeredCheckpointTest